ui/src/App.vue

```diff
@@ -12,7 +15,6 @@
 import axios from 'axios';
 
 import SearchBar from './components/SearchBar';
-// import CardList from './components/CardList';
 import CardListItem from './components/CardListItem';
 
 const mockAPI = "https://5eb817ae5652960016785cf7.mockapi.io/api/v1/analytics" // analytics, postings
```

Comment:
You need to get sensitive information and/or configuration value from environment variable or config files, instead of including in the source code

Reply:
Created .env file to store api to environment variable